Header header
string robot_name
float32 x_coord
float32 y_coord
